lib/pref.js

```diff
@@ -16,9 +16,11 @@ var pref = function(){
 		singleClickShift: false,
 		singleClickAlt: true,
 		urlMap: "",
+		useCache: false,
```

Comment:
Maybe it should default to true? I think it would send 2 requests for uncached images. See if we can avoid it.